views/layout.hbs

```diff
@@ -1,6 +1,30 @@
 <!DOCTYPE html>
 <html>
   <head>
+
+    <div>
+      {{!-- our logo and title --}}
+      <nav>
+      <form action="/movies/search" method="POST">
+    <input type="text" name="search" placeholder="Search you Movie here!">
+    <button type="submit">Search</button>
+</form>
+      <ul>
+        <li><a href="/"> Home</a></li>
+        <li>{{#if user}}
```

Comment:
the if is working the home page, but not on search (it's showing as if the user wasn't logged in but we can see it later)